.github/workflows/build.yml

```diff
@@ -32,10 +32,10 @@ jobs:
         pip install .
         pip install nox
 
-    - name: Run test
-      run: nox -s test-${{ matrix.python-version }}
+    - name: Run configuration test
+      run: nox -s configuration-${{ matrix.python-version }}
```

Comment:
is there a task missing here to run the example tests as part of the CI?

Reply:
We should not run these tests as part of CI.